src/browsing.md

```diff
@@ -234,37 +237,38 @@ parts of the browser, and to go up and down the card list.
 
 ## Find and Replace
 
-To replace text in selected notes, you can either:
+This dialog allows for replacing text on notes. As described above, it is available
+from the toolbar and the table's context menu.
 
-- Right click on selected notes in the _card list_, and click **Find and Replace**.
+The first input field is for the text that is going to be replaced, the second
+one for the replacement. Next, there is a dropdown menu that allows you to specify
+where Anki should look for text to replace: in a note's tags (requires Anki 2.1.45+),
+in all fields, or just in a specific field (only fields belonging to a selected
+note will be listed).
 
-- In the **Browser** window, click **Notes** in the menu bar and click **Find and Replace**.
+By default, only selected notes will be affected. If you want to lift that
+restriction, you can untick the first checkbox (requires Anki 2.1.45+).
```

Comment:
```suggestion
restriction, you can untick the "selected notes only" checkbox (requires Anki 2.1.45+).
```

Perhaps naming the checkbox instead of referring to its position is a bit less likely to fall out of sync when changes are made to the interface in the future?